### ISSUE TITLE
Autocomplete for variables

### ISSUE THREAD

**Issue by aiboy** [discoveryjs/discovery#7]
Auto complete dons't work for variables with helper
```
$a: <($.a.b)>;

[{ a: { b: 2 } }].exec($a)
```
helper
```
exec(...args) {
      let ctx = args.shift();
      return args.reduce((ctx, fn) => {
        if (typeof fn === 'function'){
            console.log('fn');
          return fn(ctx);
        }
        throw 'Unsupproted argument ' + fn + ' of type: ' + typeof fn;
      }, ctx);
    }
```
you can try to remove ".b"; at first line and write "." - Autocomplete won't show up

**Comment by lahmatiy** [comment]
I found that in tollerant mode jora inserts suggestion point (symbol) before a function that cause to parse error. Need to fix jora’s tollerant parsing.

**Comment by lahmatiy** [comment]
Fixed by bumping jora in `1.0.0-beta.10`
